packages/medusa/src/api-v2/admin/invites/[id]/route.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  ContainerRegistrationKeys,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import { MedusaRequest, MedusaResponse } from "../../../../types/routing"
+import { deleteInvitesWorkflow } from "@medusajs/core-flows"
+import { IUserModuleService, UpdateUserDTO } from "@medusajs/types"
+import { ModuleRegistrationName } from "../../../../../../modules-sdk/dist"
+
+// Get invite
+export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
+  const { id } = req.params
+
+  const moduleService: IUserModuleService = req.scope.resolve(
+    ModuleRegistrationName.USER
+  )
+  const invite = await moduleService.retrieveInvite(id, req.retrieveConfig)
```

Comment:
suggestion: Use remoteQuery

Reply:
we will have to migrate to remote query indeed, maybe another pr to not block